### PR TITLE
WIP: new vendor ide version support

### PR DESCRIPTION
**Pull request by yrabbit**
The .tm file is not being processed

Signed-off-by: YRabbit <rabbit@yrabbit.cyou>

### CODE REVIEW

**Comment by yrabbit** [comment]
Still most of the I/O modes are disabled, not because they won't work, but to speed up base generation for CI debugging purposes.

File: .github/workflows/chipdb.yml [hunk 1]
```diff
@@ -9,67 +9,82 @@ jobs:
...
+          path: apycula/GW1NS-2.pickle
+    - name: Archive stage artifact
+      uses: actions/upload-artifact@v2
+      with:
+          name: gw1ns-2-stage
+          path: GW1NS-2*
   pypi:
     runs-on: ubuntu-latest
-    needs: [gw1n9, gw1n1, gw1n4, gw1ns2]
+    needs: [gw1n1, gw1n9, gw1n4, gw1ns2]
```
Comment (pepijndevos) [inline .github/workflows/chipdb.yml:80]: what's all this reshuffling about??

File: apycula/clock_fuzzer.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ def dff(mod, cst, row, col, clk=None):
     dff.portmap['Q'] = name+"_Q"
     mod.wires.update(dff.portmap.values())
     mod.primitives[name] = dff
-    cst.cells[name] = f"R{row}C{col}"
+    cst.cells[name] = (row, col, 0, 'A') # f"R{row}C{col}"
```
Comment (pepijndevos) [inline apycula/clock_fuzzer.py:25]: did I do this.... well if it works it works haha

File: apycula/tiled_fuzzer.py [hunk 3]
```diff
@@ -152,87 +152,88 @@ def is_illegal(pin, attr):
 
 iostd_drive = {
             ""            : ["4", "8", "12"],
-            "LVTTL33"     : ["4", "8", "12", "16", "24"],
+#            "LVTTL33"     : ["4", "8", "12", "16", "24"],
```
Comment (pepijndevos) [inline apycula/tiled_fuzzer.py:155]: these probably need to be uncommented again

File: .github/workflows/chipdb.yml (hunk 1 above)
Comment (yrabbit) [reply .github/workflows/chipdb.yml:80]: my bad: GW1N-1 is always being experimented on, so sometimes it's just one in the list.

**Comment by uis246** [comment]
Yay. GW1NSR-4C next?

**Comment by pepijndevos** [comment]
It's possible. At this point it's probably just copy-pasting a few lines. But there are also some other high priority features and bugs so we'll see.

If you want to give it a go yourself, have a look at https://github.com/YosysHQ/apicula/commit/fef98e568a9605f45f2f76407cce693a92acd219 for an example.

**Pull request merged**